conclusion_and_future_work.tex

```diff
@@ -0,0 +1,12 @@
+SymPy is a robust CAS that provides a wide array of features. It is written in
+a general purpose programming language, Python, which allows it to be used in
+a first-class way with other Python projects, including the scientific Python
+stack. It is designed to be used in an extensible way. Unlike many other CASs,
+it is designed to be used both as a end-user application and as a library.
+
+The goal of SymPy is somewhat lofty: to be a full-featured CAS\@. However, a
+few things sit at the head of the project's roadmap.
+
+% Performance. Mention SymEngine.
```

Comment:
Feel free to push stuff up to this branch, and merge it when you think it is ready.